presentation/chapters/de-DE/mutability.chapter

```diff
@@ -0,0 +1,52 @@
+# Mutabilität
+
+---
+
+Variablen sind standardmäßig immutabel, können aber mutabel gemacht werden.
+
+```rust
+let some_immutable = true;
+let mut some_mutable = false;
+some_mutable = true;
+```
```

Comment:
Ich würde bevorzugen, wenn die Beispiele in "main()" stehen, dann lassen sie sich ausführen.

Hier kann auch mal ein Beispiel stehen, dass nicht kompiliert.